v2/devices/FP4.yml

```diff
@@ -1,5 +1,5 @@
 name: "Fairphone 4"
-codename: "FP4"
+codename: "fp4"
 formfactor: "phone"
 aliases: ["fp4"]
```

Comment:
If you change codename from `FP4` to `fp4` you may want to update that alias to be `FP4` or drop it alltogether.